www/src/content/docs/home/get-started.md

```diff
@@ -0,0 +1,45 @@
+---
+title: FPX Documentation
+description: entry point for the fpx docs site
+---
+
+FPX is a local debugging companion designed to help you to build and explore your [Hono](https://www.hono.dev) APIs. 
+
+## Get started
+### 1. Setup a Hono project
+You can set up a basic Hono project via your terminal:
+```
+ npm create hono@latest my-app 
+ ```
+
+
+If you want a project including an ORM, a database and already has a cloud deployment platform in mind we recommend using [HONC](https://honc.dev/):
+
+```
+create honc-app@latest 
+```
+
```

Comment:
also need to install middleware:

npm i@fiberplane/hono-otel

((maybe needs specific version/tag))

Reply:
updated in the docs. We haven't published the npm package yet right?

Reply:
it's up! Alpha version and no proper readme yet, but it's a prio for this week for us: https://www.npmjs.com/package/@fiberplane/hono-otel